src/command-handlers/configure.ts

```diff
@@ -72,7 +72,7 @@ export const configureUserPresenceVerification = async (options: {
 
     if (method === 'biometrics') {
         if (process.platform === 'darwin') {
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-var-requires
+            // eslint-disable-next-line @typescript-eslint/no-var-requires
             const nodemacauth = require('node-mac-auth') as typeof import('node-mac-auth');
```

Comment:
What a strange construct :)

Reply:
it's done like that because esbuild is too dumb to understand the dynamic imports